src/sentry/conf/server.py

```diff
@@ -1377,3 +1378,6 @@ def get_raven_config():
 JS_SDK_LOADER_SDK_VERSION = ''
 # This should be the url pointing to the JS SDK
 JS_SDK_LOADER_DEFAULT_SDK_URL = ''
+
+# too much junk happening from bots, lets kill things that clearly shouldn't be humans
+INVALID_EMAIL_ADDRESS_PATTERN = re.compile(r'^\d\@qq\.com$', re.I)
```

Comment:
```suggestion
INVALID_EMAIL_ADDRESS_PATTERN = re.compile(r'^\d+\@qq\.com$', re.I)
```

I think you want this, otherwise it won't work for anything I've seen.